orcestra_book/plans/HALO_20240919a_plan.py

```diff
@@ -0,0 +1,125 @@
+''' helper functions for rendering HALO-20240919a flight plan'''
+
+def geod_azi(latlon1, latlon2):
+    import pyproj
+    geod = pyproj.Geod(ellps="WGS84")
+
+    return geod.inv(latlon1.lon, latlon1.lat, latlon2.lon, latlon2.lat)[0]
+
+def geod_dist(latlon1, latlon2):
+    import pyproj
+    geod = pyproj.Geod(ellps="WGS84")
+
+    return geod.inv(latlon1.lon, latlon1.lat, latlon2.lon, latlon2.lat)[2]
+
+def centre_from_tangent(latlon1, latlon2, radius, add_angle=0):
+    import pyproj
+    geod = pyproj.Geod(ellps="WGS84")
+ 
+    az12, az21, dist = geod.inv(latlon1.lon, latlon1.lat, latlon2.lon, latlon2.lat)
+    azi_angle = az12 + add_angle
+
+    return latlon2.course(azi_angle, radius)
+
+def plot_satellite_track(ax, lon, lat, c, ls, label):
+
+    ax.plot(lon, lat, c=c, ls=ls, label=label)
+
+    for i in range(1, len(lon), 10):  # Adjust the step to control the number of arrows
+        ax.annotate('', xy=(lon[i], lat[i]), xytext=(lon[i-1], lat[i-1]),
+                    arrowprops=dict(arrowstyle="->", color=c))
+
+def plot_forcast_overlay(fig, ax, ifs_ds, ifs_fcst_time, title_label, plot_func, plot_func_kwargs):
+    from datetime import datetime
+
+    plot_func(fig, ax, ifs_ds, **plot_func_kwargs)
+
+    ifs_fcst_time_str = ifs_fcst_time.astype(datetime).strftime("%Y-%m-%d %H:%M:%S")
+    title = ax.get_title()+f"\n({title_label} forecasted on {ifs_fcst_time_str})"
+    ax.set_title(f"{title}", fontsize=15, y=1.1)
+
+def plot_atc_zones(ax, worldfirs_json):
```

Comment:
Note that since the latest release, there's `orcestra.flightplan.plot_fir()` available, which probably should be used instead.